<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Phenotype_select</class>
 <widget class="QDialog" name="Phenotype_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phenotype select</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QPushButton" name="add_file_btn">
     <property name="text">
      <string>Add File...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="phenotype_ds_gbox">
     <property name="minimumSize">
      <size>
       <width>191</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Phenotype Discrete? (Select FIRST!)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="discrete_rbtn">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Discrete</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">phenotype_ds_gp</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="scalar_rbtn">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Scalar</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">phenotype_ds_gp</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTextBrowser" name="neg_txt">
     <property name="placeholderText">
      <string>Negative discrete phenotypes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTextBrowser" name="pos_txt">
     <property name="placeholderText">
      <string>Positive discrete phenotypes</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="pos_add_txt">
     <property name="placeholderText">
      <string>Single discrete phenotype</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pos_add_btn">
     <property name="text">
      <string>Add as positive</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pos_rmv_btn">
     <property name="text">
      <string>Revert as negative</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="fin_btn">
     <property name="text">
      <string>Finalize phenotype (Patience...)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="phenotype_ds_gp"/>
 </buttongroups>
</ui>
